docker/Dockerfile.gpu

```diff
@@ -26,12 +28,18 @@ RUN apt-get update -y && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-# RUN pip3 install -U pip setuptools>=41.0.0
-RUN python3 -m pip install -U pip setuptools && python3 -m pip install --no-cache-dir \
+#Install python3 and setuptools
+RUN python3 -m pip install -U pip setuptools
+
+#Install custom tensorflow
+RUN python3 -m pip install --no-cache-dir \
```

Comment:
Better to install the custom tensorflow after the rest; or else you might get dependencies overwritten.